stack/deprecating-interfaces.rst

```diff
@@ -6,9 +6,8 @@ Deprecation Procedure
 =====================
 
 Deprecations are changes to interfaces.
-(If they were limited to implementation alone, they wouldn't require deprecation, as no one would notice when the change was made.)
-As a result they often require or result from :doc:`RFCs </communications/rfc>`.
-But our usual policy applies; if no one would object, a deprecation can be made without an RFC.
+(If they were limited to implementation alone, they wouldn't require deprecation, as no one would notice when the change was made)
+They require approval from the DMCCB by means of an :doc:`RFC </communications/rfc>`.
```

Comment:
I'm a little worried about excessive bureaucracy here, but I don't think it's that big a deal in practice.